front/components/home/new/menu/config.ts

```diff
@@ -91,6 +91,9 @@ const MoreMenuConfig: MenuConfig = {
     },
     LegalMenuConfig.items[0],
     LegalMenuConfig.items[1],
+    {
+      title: " ",
```

Comment:
??

Reply:
Ugly but works ^^'

Reply:
To do what?

Reply:
<img width="674" alt="image" src="https://github.com/dust-tt/dust/assets/4435185/38bad0d6-78e9-414e-9bf0-805a8fd5a81c">
The empty space right there